cmd/podman/compose.go

```diff
@@ -170,6 +170,21 @@ func composeEnv() ([]string, error) {
 	}, nil
 }
 
+// composeShouldLogWarning returns whether a notice on engine redirection should be piped to stderr regardless of logging configuration
+func composeShouldLogWarning() bool {
+	if shouldWarnLogsEnv, ok := os.LookupEnv("PODMAN_COMPOSE_WARNING_LOGS"); ok {
+		if shouldWarnLogsEnvVal, err := strconv.ParseBool(shouldWarnLogsEnv); err != nil {
+			return shouldWarnLogsEnvVal
+		}
+	}
+	if shouldWarnLogsEnv, ok := os.LookupEnv("COMPOSE_WARNING_LOGS"); ok {
+		if shouldWarnLogsEnvVal, err := strconv.ParseBool(shouldWarnLogsEnv); err != nil {
+			return shouldWarnLogsEnvVal
+		}
+	}
```

Comment:
This doesn't make sense to me, there is no benefit for two different env vars for the same thing

Reply:
>I'm thinking a similar flag could be proposed for docker-compose and it would be valuable with one that is agnostic of the tool.

I don't see the point of this, the warning is printed by podman (not docker-compose or podman-compose) to tell users that we just execute another program and do not have compose support ourself.

So yes I like to remove this var.